### DISCUSSION TITLE
Вопросы по API

### DISCUSSION THREAD

**Discussion by NZhukovaITBI** [totumonline/totum-ru-issues-and-discussions#389] [Как это сделать?] [answered]
Добрый день,
спасибо за столь замечательный продукт!
Помогите пожалуйста с JSON запросом:
1. Как обратиться к полю сортировки (order_field) ? что бы не дублировать его в отдельном поле, или как вариант сдублировать в отдельное поле значения order_field и уже к нему обращаться через запрос, обращение через #n не работает. https://ru.docs.totum.online/tables#table-settings-order_field
3.  Есть таблица с числовыми значениями, как в колонку проставить 0 для всех строк через JSON запрос, для примера нужно обнулить все строки в колонке "Факт"
Заранее спасибо огромное!
![totum](https://github.com/totumonline/totum-ru-issues-and-discussions/assets/133749219/b773fc5c-779a-4237-a950-6db62c885df1)

**Comment by erpic-ru** [comment, marked as answer]
Можно через Remotes интегрироваться, например. Там точно все будет работать

Второй случай вроде как рассмотрен в документации:

```
{
  "import": {
    "rows-set-where": [
      {
        "where": [
          {
            "field": "fact",
            "value": "0",
            "operator": "!="
          }
        ],
        "set": {
          "fact": 0
        }
      }
    ]
  }
}

```

**Reply by ttm-alexeypolunin** [reply]
Нам как разрабам JSON-API не очень нравится. Мы считаем Remotes более прогрессивными тк там прям на тотум-коде можно написать че над сделать.

**Reply by ttm-alexeypolunin** [reply]
#n доступен, когда включен параметр - сортировка по n-полю

**Reply by NZhukovaITBI** [reply]
Спасибо огромное за помощь! если еще будут идеи по поводу order_field -  пишите ! :)
Через remotes - не удобный вариант, проще свое поле сделать

**Reply by NZhukovaITBI** [reply]
Спасибо ! Все получилось ! Вы лучшие !
